ClosedXML/Utils/ColorStringParser.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Drawing;
+using System.Globalization;
+
+namespace ClosedXML.Utils
+{
+    internal static class ColorStringParser
+    {
+        public static Color ParseFromHtml(string htmlColor)
+        {
+            try
+            {
+                return ColorTranslator.FromHtml(htmlColor);
+            }
+            catch
+            {
+                // https://github.com/ClosedXML/ClosedXML/issues/675
+                // When regional settings list separator is # , the standard ColorTranslator.FromHtml fails
+                return Color.FromArgb(int.Parse(htmlColor.Replace("#", ""), NumberStyles.AllowHexSpecifier));
```

Comment:
This would give a wrong result with FF00FF (Alpha will be 0 instead of 255, that is used by ColorTranslator by default)  or with FFF (in HTML it is equal to FFFFFFFF). I doubt this will be an issue in ClosedXML since it gets values stored in a file. Just FYI.

Reply:
Yeah, you're right, but I'm not going to spend too much time on such an edge case. Merging this as is.